pages/common/popeye.md

```diff
@@ -0,0 +1,20 @@
+# popeye
+
+> Utility that reports potential issues with your Kubernetes deployment manifests and configuration.
+> Homepage: <https://github.com/derailed/popeye>.
+
+- Scan your Kubernetes cluster:
```

Comment:
Perhaps this will read better?
```suggestion
- Scan the current Kubernetes cluster:
```

How does it know what cluster to scan?

Reply:
You always have a default k8s cluster based on `~/.kube/config`

Reply:
btw, the suggestion is good.

Reply:
Ah, I see. Thanks!